test/lit/validation/custom-descriptors.wast

```diff
@@ -0,0 +1,19 @@
+;; Test that custom descriptors instructions are validated correctly.
+
+;; RUN: not wasm-opt %s -all 2>&1 | filecheck %s
+
+;; CHECK: [wasm-validator error in function ref.get_desc-inexact] function body type must match, if function returns
+
+(module
+  (rec
+    (type $described (descriptor $describing (struct)))
+    (type $describing (describes $described (struct)))
+  )
+
+  (func $ref.get_desc-inexact (param (ref null $described)) (result (ref (exact $describing)))
+    ;; The result should be inexact because the input is inexact.
+    (ref.get_desc $described
+      (local.get 0)
+    )
```

Comment:
Is this not tested by

https://github.com/WebAssembly/binaryen/pull/7614/files#diff-adc983d694edd27a2331cfe3dbbf0c82753c5bba63db196e064c3a28db7d5dd2R75

?

Reply:
This is testing the converse. If there were a bug and the result of `ref.get_desc` were always exact, that other test would still pass.